app/cafe.py

```diff
@@ -0,0 +1,27 @@
+import datetime
+from app.errors import NotVaccinatedError
+from app.errors import OutdatedVaccineError
+from app.errors import NotWearingMaskError
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        try:
+            if not visitor["vaccine"]:
+                raise NotVaccinatedError("visitor is not vaccinated")
+        except KeyError:
+            raise NotVaccinatedError("visitor is not vaccinated")
+        try:
+            if visitor["vaccine"]["expiration_date"] < datetime.date.today():
+                raise OutdatedVaccineError("visitors vaccine is outdated")
+
+            if visitor["wearing_a_mask"] is False:
+                raise NotWearingMaskError("visitor should wear a mask")
+
+        except KeyError as e:
+            raise NotVaccinatedError(f"missing key: {e}")

```

Comment:
The exception message for missing keys should be more descriptive. Currently, it raises `NotVaccinatedError` with a message indicating a missing key, which might be misleading. Consider using a more appropriate error message or exception type for missing keys.